recipes/scikit-mobility/meta.yaml

```diff
@@ -0,0 +1,122 @@
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive: 
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  sha256: 32d26bff82a1403a2e05deba59045fe7706b73eea8db4d0083ec941ad79f512c
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - folium
+    - geojson
+    - geopandas
+    - matplotlib
+    - numpy
+    - pandas
+    - pip
+    - powerlaw
+    - python
+    - requests
+    - rtree
+    - scikit-learn
+    - scipy
+    - shapely
+    - statsmodels
+    - tqdm
+  run:
+    - folium
+    - geojson
+    - geopandas
+    - matplotlib
+    - numpy
+    - pandas
+    - powerlaw
+    - python
+    - requests
+    - rtree
+    - scikit-learn
+    - scipy
+    - shapely
+    - statsmodels
+    - tqdm
+
+test:
+  imports:
+    - skmob
+    - skmob.core
+    - skmob.io
+    - skmob.measures
+    - skmob.models
+    - skmob.preprocessing
+    - skmob.privacy
+    - skmob.tessellation
+    - skmob.utils
+  requires:
+    - pytest
+
+about:
+  home: "https://github.com/scikit-mobility/scikit-mobility"
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Use the SPDX identifier, e.g: GPL-2.0-only instead of GNU General Public License version 2.0
+  # See https://spdx.org/licenses/
+  license: BSD-3-Clause
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: BSD
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
+  license_file: LICENSE.txt
```

Comment:
I opened an issue upstream about manually packaging the license file:

https://github.com/scikit-mobility/scikit-mobility/issues/137

Reply:
Thanks! We'll add this in the new release.